### PR TITLE
docs: update AudioCache explanation, migration guide, replace package READMEs

### LINKED ISSUE #1256: docs:  more descriptive codes and information in Getting Started tutorial for AudioCache
## Checklist
- [x] I made sure that the feature I am proposing doesn't already exist

## Proposal / Solution

<details>
  <summary>Example Code</summary>

  

  ```dart
  void main() {
  }
  ```
</details>

## Platforms

Affected Platforms:

### PR DESCRIPTION
**Pull request by Gustl22**
# Description

* update AudioCache explanation (closes #1256)
* replace package README.md symlinks with files, and switched main README file to `audioplayers` package (#1433, https://github.com/dart-lang/pub/issues/2699)
* update migration guide to use Changelog
* update contribution guide
* update parity table

Copying Symlink targets for release should not be necessary anymore.
 
## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:`, `chore:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation and added dartdoc comments with `///`, where necessary.
- [ ] I have updated/added relevant examples in [example].

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Review by spydon — APPROVED** [review]

File: README.md
Comment (luanpotter) [inline README.md:1]: I'm hoping github will understand a symlink

File: packages/audioplayers/README.md
Comment (luanpotter) [inline packages/audioplayers/README.md:1]: we should update the Support section based on the flame repo

File: packages/audioplayers_android/README.md
Comment (luanpotter) [inline packages/audioplayers_android/README.md:1]: do we need to escape single underscore? I don't think I've ever seem that on other markdowns

**Review by luanpotter — APPROVED** [review]
LGTM

File: README.md
Comment (Gustl22) [reply README.md:1]: Seems like GH is showing it fine also for root, or do you have a special case in mind?
https://github.com/bluefireteam/audioplayers/tree/gustl22/update-docs

File: packages/audioplayers/README.md
Comment (Gustl22) [reply packages/audioplayers/README.md:1]: Updated, but don't know if you want the badge also, and if want to credit community somewhere.

File: packages/audioplayers_android/README.md
Comment (Gustl22) [reply packages/audioplayers_android/README.md:1]: Saw it here, but don't think it's needed: https://raw.githubusercontent.com/flutter/packages/main/packages/video_player/video_player_avfoundation/README.md

**Comment by Gustl22** [comment]
@luanpotter @spydon I also now updated some of the contributing stuff (regarding the lack docs for setting up Melos, which is quite important I think). I know that there could be updated much more, like in flame. But I also think every new passage needs to be maintained. So I'd rather keep it simple and leave out the redundant stuff.

**Review by spydon — APPROVED** [review]

**Pull request merged**